config/settings.yml

```diff
@@ -1283,6 +1283,7 @@
         :poll: 10.seconds
       :event_catcher_swift:
         :poll: 10.seconds
+      :event_catcher_openstack_service: "auto"
```

Comment:
Is there anyway this can be bumped up to the `:event_catcher_openstack` section?

I'd rather have it there, than add another new section for OpenStack.  Thoughts?